### PR TITLE
[FIX] Fixed svg function to return svg chart together with container div for highcharts

### PR DESCRIPTION
**Pull request by PrimozGodec**
It is required that also axis labels are provided in a report.  

Also fixed svg function in webview to take return whole svg. Before svg was returned without last `>`

### CODE REVIEW

**Comment by PrimozGodec** [comment]
@kernc It is corrected now.

File: Orange/widgets/highcharts.py [hunk 1]
```diff
@@ -325,6 +325,12 @@ def _on_selected_points(self, points):
         self._selection_callback([np.sort(selected).astype(int)
                                   for selected in points])
 
+    def svg(self):
+        """ Return SVG string of the first SVG element on the page, or
+        raise ValueError if not any. """
```
Comment (kernc) [inline Orange/widgets/highcharts.py]: Add a comment why this is overridden and why a non-SVG is returned.

Comment (kernc) [inline Orange/widgets/highcharts.py]: Please add a comment why this is overridden and why a non-SVG is returned.

**Comment by PrimozGodec** [comment]
Done.

**Pull request merged**